delfin/api/v1/storages.py

```diff
@@ -217,4 +217,5 @@ def _set_synced_if_ok(context, storage_id, resource_count):
                 storage['sync_status'] > 0:
             raise exception.StorageIsSyncing(storage['id'])
         storage['sync_status'] = resource_count * constants.ResourceSync.START
+        storage['updated_at'] = current_time
```

Comment:
oslo_db package will  take care of updating the updated_at to the current time? do we need to explicitly set this?

Reply:
Yes you are right, but there is one scenario that when the `interval` from line 211 if bigger than CONF.sync_task_expiration, when we send sync request, it would start sync task no matter what the `sync_status` is, which is what we expected. But if we send another sync request in a short time after the first sync request, because all the info (storage/pool/volume) has not been updated, so the `updated_at` also has not been updated by oslo_db, so the `interval` from line 211 is still bigger than CONF.sync_task_expiration, so no matter what the sync status is(line 216), the second sync task would be lauched, which was not expected.
So, we need to update the `update_at` manually when we update the `sync_status` to launch the sync task.

Reply:
Ok got it.. thanks